lib/sigh/developer_center.rb

```diff
@@ -320,7 +318,7 @@ def create_profile(app_identifier, type, cert_date)
     end
 
     def renew_profile(profile_id, type, cert_date)
-      certificate = code_signing_certificate(type, cert_date)
+      certificate = code_signing_certificates(type, cert_date).first
```

Comment:
Also, I don't like this. Though I'm not sure which page is this, where we have only radio inputs.